codersdk/workspaceagents.go

```diff
@@ -340,6 +340,39 @@ func (c *Client) ListenWorkspaceAgent(ctx context.Context) (net.Conn, error) {
 		return nil, readBodyAsError(res)
 	}
 
+	go func() {
+		tick := 30 * time.Second
```

Comment:
We should add comments here. I could see someone removing this because they believe it's unnecessary.